posts/math-libraries-intro/README.md

```diff
@@ -0,0 +1,128 @@
+# GEMM Examples
+
+============================
+
+Matrix Multiplication performed using OpenBLAS and cuBLAS.
+
+## Getting Started
+
+============================
+
+### Packages Used
+
+- CUDA Toolkit 11.0
```

Comment:
Why so old?

Reply:
We tested it on 11.3, so I changed it to that. Just wanted to show users what we used to run these examples